src/quo2/components/markdown/text_input.cljs

```diff
@@ -0,0 +1,270 @@
+(ns quo2.components.markdown.text-input
```

Comment:
please don't copy old components, we should reimplement them instead without using quo or status-im namespaces

Reply:
So should I refactor them to not using `quo1` or `status-im` ?
Making them independent of these two namespace ?